src/components/views/gifpicker/GifPicker.tsx

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable camelcase */
+import React from "react";
+import { replaceableComponent } from "../../../utils/replaceableComponent";
+import Search from "../emojipicker/Search";
+import ScrollPanel from "../../structures/ScrollPanel";
+import Spinner from "../elements/Spinner";
+import GifThumbnail from "./GifThumbnail";
+import { DebouncedFunc, throttle, uniqBy } from "lodash";
+import { Gif } from "./Gif";
+
+const API_KEY = "hhMBhTnD7k8BgyoZCM9UNM1vRsPcgzaC";
```

Comment:
This probably shouldn't be checked into git. 
Unrelated though, this work is amazing :)

Reply:
Thanks!

This is a public key so it is not a security issue, as the request will be done client-side and anyone can inspect it.
https://github.com/Giphy/giphy-js/issues/120#issuecomment-680988917

If this is approved by the product team a production key is needed in order to increase the limits by Giphy.